packages/api/src/taskInfo.ts

```diff
@@ -17,7 +17,8 @@
  ***********************************************************************/
 
 export type TaskState = 'running' | 'completed';
-export type TaskStatus = 'in-progress' | 'success' | 'failure' | 'canceled';
+export const TASK_STATUSES = ['in-progress', 'success', 'failure', 'canceled'] as const;
```

Comment:
Why not sing an enum ? This synthax seems a bit complicated, why do we need to iterate on the statuses anyway?

Reply:
enum = it'll break everything (all the code using TaskStatus)

> why do we need to iterate on the statuses anyway?

to display tabs to filter status. There is a need of one tab per status (so iterating)